cldf/README.md

```diff
@@ -88,7 +88,7 @@ Name/Property | Datatype | Description
 property | value
  --- | ---
 [dc:conformsTo](http://purl.org/dc/terms/conformsTo) | [CLDF ParameterTable](http://cldf.clld.org/v1.0/terms.rdf#ParameterTable)
-[dc:extent](http://purl.org/dc/terms/extent) | 191
+[dc:extent](http://purl.org/dc/terms/extent) | 210
 
```

Comment:
hmm this should be 191 -- any idea why the badwords list isn't propagating here?

Reply:
resolved